pkgs/applications/networking/eddie-ui/default.nix

```
@@ -0,0 +1,92 @@
+{ stdenv,
+  fetchFromGitHub,
+  clang,
+  cmake,
+  mono,
+  dotnet-sdk,
+  which,
+  glibc,
+  pkgconfig,
+  gtk2,
+  libappindicator-gtk2,
+  lzma,
+  libselinux,
+  libsepol,
+  fribidi,
+  libthai,
+  libdatrie,
+  xorg,
+  openvpn,
+  stunnel
+  }:
```

```suggestion
{ stdenv
, fetchFromGitHub,
,  clang
,  cmake
....
}:
```

I am lazy now but the formatting should be like the above example.